### PR TITLE
French body names

### PR DESCRIPTION
**Pull request by eggrobin**
Very minimalistic support for French (I did not bother with the descriptions and biome names), but since the body names appear throughout the UI—both that of stock and that of other mods—this should be a significant improvement.

Note the comment ll. 21 sqq.; we are misusing Lingoona, but we have to if we want to avoid breaking the stock UI.

### CODE REVIEW

File: GameData/RealSolarSystem/Localization/fr-fr.cfg [hunk 1]
```diff
@@ -0,0 +1,95 @@
+Localization
+{
+    fr-fr
+    {
+        //Planet & Biomes
+
+        // The genders of celestial bodies that do not have articles (that is,
+        // all but « le Soleil », « la Terre », and « la Lune ») are somewhat
+        // unclear.  From the gender of their namesakes, it appears to drift
```
Comment (pleroy) [inline GameData/RealSolarSystem/Localization/fr-fr.cfg:9]: As discussed offline, Littré (1863-1873) uses systematically the gender of the eponymous divinity.  I would stick to that usage, as it's as good a linguistic reference as any other.  Arguably the usage is fluctuating, but we have to choose a normative reference.  FWIW, personally I find `Cérès^M` and `Jupiter^F` strange.

File: GameData/RealSolarSystem/Localization/fr-fr.cfg [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        #RSS_Oberon_name = Oberon^M
+
+        #RSS_Miranda_name = Miranda^M
+
+        #RSS_Venus_name = Venus^F
+
+        #RSS_Vesta_name = Vesta^F
+    }
+}
```
Comment (pleroy) [inline GameData/RealSolarSystem/Localization/fr-fr.cfg]: EOL at the end.

File: GameData/RealSolarSystem/Localization/fr-fr.cfg (hunk 1 above)
Comment (eggrobin) [reply GameData/RealSolarSystem/Localization/fr-fr.cfg:9]: This makes most easier, but then we have the moons of Uranus.

**Review by pleroy — APPROVED** [review]

File: GameData/RealSolarSystem/Localization/fr-fr.cfg [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+        #RSS_Callisto_name = Callisto^F
+
+        #RSS_Mars_name = Mars^M
+
+        #RSS_Phobos_name = Phobos^M
+
+        #RSS_Deimos_name = Deimos^M
+
+        #RSS_Mercury_name = Mercury^M
```
Comment (eggrobin) [inline GameData/RealSolarSystem/Localization/fr-fr.cfg:53]: Par Mercure !

**Pull request merged**